dev/benchmarks/macrobenchmarks/test_driver/frame_policy_test.dart

```diff
@@ -0,0 +1,58 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
This file is modified from https://github.com/flutter/plugins/blob/master/packages/e2e/lib/e2e_driver.dart